eng/Version.Details.xml

```diff
@@ -21,9 +101,17 @@
       <Uri>https://github.com/dotnet/coreclr</Uri>
       <Sha>20904a2629729e33531f30d87ae04b285cb34ee7</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Private.WindowsDesktop.App" Version="3.0.0-preview6-27702-36">
-      <Uri>https://dev.azure.com/devdiv/DevDiv/_git/DotNet-Trusted</Uri>
-      <Sha>45ef76ae65decfab78855206936a12b00d9dd87a</Sha>
+    <Dependency Name="Microsoft.Private.Winforms" Version="4.8.0-preview6.19251.3">
+      <Uri>https://github.com/dotnet/winforms</Uri>
+      <Sha>084316b6d123706f5e2e075f5bdf2e08f0a69f9a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Wpf.GitHub" Version="4.8.0-preview6.19251.2">
```

Comment:
I think I'd want this to day `CoherentParentDependency="Microsoft.Private.Winforms"`

/cc @mmitche, @zsd4yr

Reply:
I think this is backward. Because winforms flows into wpf, you want the winforms version to be tied to what build got pulled into wpf

Reply:
Oh I see! Yes ignore my earlier comment. 

So we want wpf.github would be the coherency parent of Winforms  and then wpf.dnceng would be the parent of wpf.github ?

Reply:
`wpf.dnceng` depends on both `wpf.github` and `winforms` directly (in repo-to-repo terms), so for now I made `wpf.dnceng` the parent of both `wpf.github` and `winforms`. Unclear if I need to point to the highest-level dependency, can't find any docs on what `CoherentParentDependency` is actually doing.

Reply:
@mmitche @vatsan-madhavan @zsd4yr any objection to the setup I have in this PR now?

```xml
<Dependency Name="Microsoft.Private.Winforms" ... CoherentParentDependency="Microsoft.DotNet.Wpf.DncEng">
<Dependency Name="Microsoft.DotNet.Wpf.GitHub" ... CoherentParentDependency="Microsoft.DotNet.Wpf.DncEng">
<Dependency Name="Microsoft.DotNet.Wpf.DncEng" ...>
```

Reply:
that looks correct to me.

Reply:
Cool, I'll go ahead with this. Can change it later if need be.